src/Application/Service/Property/PropertyViewService.js

```diff
@@ -1,19 +1,17 @@
-var PropertyDoesNotExist = require('../../../Domain/Model/Property/PropertyDoesNotExist')
+import PropertyDoesNotExist from '@Domain/Error/Property/PropertyDoesNotExist';
```

Comment:
Errors (or exceptions) are part of the Model, what do you think if we locate them into the Model folder?

Reply:
`Model/Error/Property` or `Model/Property/Error` and generic errors in `Model/Error`

btw, i don't like package names with uppercase :P

Reply:
Model/Property/Error or Model/Property/Exceptions, maybe exceptions is more accurate due it is not always an "error"